src/components/styling/cover.css

```diff
@@ -0,0 +1,64 @@
+.cover-image-wrapper {
+  position: relative;
+  width: 100%;
+}
+
+.cover-image-element {
+  width: 100%;
+  height: auto;
+}
+
+
+.cover-image-wrapper:hover .cover-image-element {
+  filter: brightness(50%);
+  transition: all 0.2s ease-in-out;
+}
+
+.icon-wrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  display: none;
+}
+
+.cover-image-wrapper:hover .icon-wrapper {
+  display: flex;
+  cursor: pointer;
+  transition: all 0.2s ease-in-out;
+}
+
+.heart-icon,
+.dots-icon {
+  width: 30px;
+  height: 30px;
+  filter: invert(0.8);
```

Comment:
is this for making the icons turn to white? good idea! We couldn't find an easy way then we change the color in svg file.

Reply:
yes, Nathalie figured it out :)